examples/Time-Series/time_series_keithley2600.py

```diff
@@ -0,0 +1,146 @@
+t_step = nplc / line_frequency  
+
+n_datapoints = int(duration / t_step + 1) # one point more since we want to measure at cero too
+i_range = 1e-3 # range of the current
+i_compliance=0.1 # max current / compliance
+
+# %% When the with-block is exited, the shutdown method of the instrument will be called, turning the system into a safe state.
+with Keithley2600("ASRL/dev/ttyUSB1::INSTR") as smu:
+
+    # First define input parameters
+    smu.reset()
+    smu.ChA.wires_mode = "2"
+    smu.ChA.source_voltage = voltage
+    sleep(0.1) # wait here to give the instrument time to react
+
+    smu.ChA.measure_current(nplc=nplc, current=i_range, auto_range=False) # with auto_range=True it takes very long and sleep(X) might to be increased
+    sleep(0.1)
+
+    # Apply bias
+    smu.ChA.apply_voltage(compliance_current=i_compliance)  # Somehow I would expect this to turn source on but it does not
+    sleep(0.1)
+    smu.ChA.source_output = "ON" # needed to turn source on
+
+    # Measure current
+    currents = np.zeros(n_datapoints)
+    times = np.zeros(n_datapoints)
+
+    for i in range(n_datapoints):
+        times[i] = time()
+        currents[i] = smu.ChA.current
+    sleep(0.1)
+    smu.ChA.source_output = "OFF" # needed to turn source off even inside with statement
+
+    # Beep once the measurement is done -> this is not implemented so we send lua strings to the smu -> for a list of commands see https://www.tek.com/keithley-source-measure-units/smu-2600b-series-sourcemeter-manual-8
+    smu.write("beeper.enable = beeper.ON")
+    smu.write("beeper.beep(.5,292.00)")
+
+    # so we actually could read the line-frequency from the device:
+    linefreq = float(smu.ask("print(localnode.linefreq)"))
+
+# %% Save the data columns in a pandas DataFrame and inspect 
+data = pd.DataFrame({
+    'Time (s)': times - times[0],
+    'Current (A)': currents
+})
+
+# For 1 MOhm and 10V bias we expect ~ 0.01 mA -> in my case it measured 0.009 mA but this should be tolerable. If your measurement is totally off check all the connections
+print(data.tail())
+print("Compare the difference of duration = {}s and the end time of {}s in our DataFrame".format(duration, data["Time (s)"][-1:].values))
+print("The timing is not accurate since there is a delaye from the for-loop. If you do only care about the number of datapoints this is not a problem")
+
+# %% Advanced example with buffer
+# At the time of writing the low level function to use the buffer of the device is not available via attributes
+# So we use the write() and read() methods to access them like the beeper above
+
+# It turns out, that the USB-serial RS-232 connection is too slow to transfer data from the buffer back to the pc for longer or faster measurements
+# Therefore we increase the baud rate. The factory-selected baud rate is 9600. Possible values = [300, 600, 1200, 2400, 4800, 9600, 19200, 38400, 57600, 115200].
+
+visa_address = list_resources()[0] # automatically receive device address
```

Comment:
Please reuse the connection string proposed above. You don't know if the first entry of that list is the correct one.